src/explore-education-statistics-admin/src/services/releaseService.ts

```diff
@@ -51,6 +51,7 @@ export interface ReleaseSummary {
   publishScheduled: string;
   nextReleaseDate?: PartialDate;
   latestInternalReleaseNote: string;
+  live?: boolean;
```

Comment:
This is optional? Would have expected it to non-optional. Sounds like a potential bug in the backend. Probably needs a ticket to sort out if so.